src/App.js

```diff
@@ -47,6 +55,28 @@ function PrivateRoute({ children, ...rest }) {
                 }}
               />
             );
+          case AuthStatus.SignedOut: {
+            switch (pwReset) {
+              case !pwReset:
+                return (
+                  <Redirect
+                    to={{
+                      pathname: '/login',
+                      state: { from: location },
+                    }}
+                  />
+                );
+              case pwReset:
+                return (
+                  <Redirect
+                    to={{
+                      pathname: '/login/pwReset',
+                      state: { from: location },
+                    }}
+                  />
+                );
+            }
+          }
```

Comment:
I don't think we need this logic. We use the login redirect to handle people that access private routes (e.g. /participants) when they aren't logged in. We send them to the login page then reload the original page after they log in. I don't think we would need to do that for password reset

Reply:
got it. Thanks for clarifying.

Reply:
Yet another update: I did a git fetch and then pull, and now my 'forgot your password' link does not navigate to the /resetPassword endpoint successfully. I commented the areas that may be the issue. Would like some help determining what's the cause of this problem here.